CMakeLists.txt

```diff
@@ -139,7 +139,7 @@ endif ()
 string(REGEX MATCH "-?[0-9]+(.[0-9]+)?$" COMPILER_POSTFIX ${CMAKE_CXX_COMPILER})
 
 find_program (LLD_PATH NAMES "lld${COMPILER_POSTFIX}" "lld")
-find_program (GOLD_PATH NAMES "gold")
```

Comment:
`find_program (… NAMES …)` accepts multiple names (see the line above).
It's better to append new name to the list.

Reply:
Indeed, fixed